src/main/java/org/folio/rest/impl/PostGobiOrdersHelper.java

```diff
@@ -44,6 +44,8 @@
 
 public class PostGobiOrdersHelper {
 
+  private static final String ORDERS_ENDPOINT = "/orders/composite-orders";
```

Comment:
I feel obsessive compulsive saying this, but it bugs me that this is above the logger declaration and not below with the other String constants.  Don't feel like you have to fix this as it doesn't really make a difference, I just couldn't resist the urge to say something.